.github/dependabot.yml

```
@@ -203,3 +203,7 @@ updates:
       interval: "daily"
     open-pull-requests-limit: 0
 
+  - package-ecosystem: 'pip'
+    directory: '/'
+    schedule:
+      interval: 'weekly'
```

We use a daily interval for all other configs here, should we do the same for pip?

```suggestion
      interval: 'daily'
```